install_config/aggregate_logging.adoc

```diff
@@ -96,14 +85,14 @@ $ oc project logging
 [NOTE]
 ====
 Specifying a non-empty
-link:../admin_guide/managing_projects.html#using-node-selectors[node
+xref:../admin_guide/managing_projects.html#using-node-selectors[node
 selector] on the project is not recommended, as this would restrict
 where Fluentd can be deployed. Instead, specify node selectors for the
-deployer to be applied on your other deployment configurations.
+deployer to be applied to your other deployment configurations.
 ====
 
-. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
-can be configured specifically for your instance of the deployer.
+. Create a xref:../dev_guide/secrets.html#[secret] for the deployer. The secret
+can be configured specifically for the deployer.
```

Comment:
Not sure there is any point in the second sentence. We ought to give some indication of what this secret is for. Something like:

"Create a xref:../dev_guide/secrets.html#[secret] to provide optional security-related files to the deployer. "